docs/content/deployment/next-steps/_index.md

```diff
@@ -261,14 +261,14 @@ The official documentation is a good resource for usage details:
   * Read about [Choria Playbooks](/docs/playbook) to write higher order orchestration scripts than the CLI provides
   * Read about [Puppet Tasks](/docs/tasks/) that provides a way to gain access to much more shared capabilities from the Puppet Forge
   * I strongly suggest you read about [using the mcollective CLI](../../concepts/cli)
-  * Read about [filtering which hosts to act on](https://docs.puppet.com/mcollective/reference/ui/filters.html)
+  * Read about [filtering which hosts to act on](https://puppet.com/docs/mcollective/current/reference/ui/filters.html)
```

Comment:
This one can go to https://choria.io/docs/concepts/cli/#selecting-request-targets-using-filters